apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -160,19 +161,28 @@ export default class LibraryPublisher extends React.Component {
     return this.props.libraryDetails.sourceFunctionList.map(sourceFunction => {
       let name = sourceFunction.functionName;
       let comment = sourceFunction.comment;
+      let shouldDisable = comment.length === 0;
+      let checked = this.state.selectedFunctions[name] || false;
+      if (shouldDisable && checked) {
+        checked = false;
+        this.setState(state => {
+          state.selectedFunctions[name] = false;
+          return state;
+        });
+      }
       return (
         <div key={name}>
           <input
             style={styles.largerCheckbox}
             type="checkbox"
-            disabled={comment.length === 0}
+            disabled={shouldDisable}
             name={name}
-            checked={this.state.selectedFunctions[name] || false}
+            checked={checked}
             onChange={this.boxChecked(name)}
           />
           <span>{name}</span>
           <br />
-          {comment.length === 0 && (
+          {shouldDisable && (
```

Comment:
Is not having a comment the only reason the checkbox is ever disabled?

Reply:
Yes

Reply:
Do you think there should be another?